recipes/emsdk/all/conanfile.py

```diff
@@ -29,17 +31,29 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("nodejs/16.3.0")
```

Comment:
There has been some talk about this recipe requiring nodejs directly, instead of it being a tool_requires, we should clarify if it's really needed or if we could get away with tool_requireing it.

The issue is that users of emsdk are not able to override nodejs, and they get conflicts using it as a tool_requires